docs/internal-documentation/maintenance/ReleaseProcess.md

```diff
@@ -62,7 +62,7 @@ listed changes are covered by MANIFEST.SKIP:
 
 For all components, generate Makefile, META.yml and others.
 
-    perl Makefile.PL
+    perl Makefile.PL --no-ed25519
```

Comment:
The `--no-ed25519` is only relevant for Zonemaster-LDNS. Have different commands or at least document that the flag is ignored for all other components.